package.json

```diff
@@ -52,7 +53,7 @@
     "watchify": "^3.7.0"
   },
   "dependencies": {
-    "popsicle": "^8.2.0",
+    "any-promise": "^1.3.0",
```

Comment:
I'd rather not add this dependency. Instead, you may provide a way to set `Promise` implementation via configuration or some other dependency injection way.